javascript/index.html

```diff
@@ -111,7 +111,8 @@ <h1 class="page-header">Intro to JavaScript: Arrays, Objects, Functions</h1>
         <li><a href="exercises/forloops.html">Exercise: for loops</a>
         <li><a href="exercises/arrays.html">Exercise: arrays</a>
         <li><a href="exercises/wordguesser.html">Homework: The Word Guesser</a>
-        <li><a href="exercises/currencyConverter.html">Homework: The Currency Converter</a>
+        <!--Link leads to nothing-->
```

Comment:
Please leave the currency converter link in. We have issue #254 to fix the broken link.